tests/integration/dropdown/test_dynamic_options.py

```diff
@@ -0,0 +1,54 @@
+import dash
+from dash.dependencies import Input, Output, State
+from dash.exceptions import PreventUpdate
+import dash.testing.wait as wait
+import dash_core_components as dcc
+
+
+def test_ddsv001_dynamic_options(dash_duo):
```

Comment:
sorry I was not clear about the suggestion. the TCID added in the previous test should be unique. more details refer to Notes #2 in example. https://dash.plot.ly/testing

this one should be sth like `dddo001`